### PR TITLE
FIX: SET Command Inconsistency

### PR DESCRIPTION
**Pull request by vanshavenger**
Screenshot 2024-10-28 at 11 09 33 PM

The above was not same in the Current Dice db implementation , i have fixed it, kindly verify once and let me know if there is something that is need to be change and updated too.

It makes it consistent with current DOCs and Redis Implementation.  
#786
cc @apoorvyadav1111

### CODE REVIEW

**Comment by JyotinderSingh** [comment]
Please add testsbto validate these changes

**Comment by vanshavenger** [comment]
Sure will do....

File: internal/eval/store_eval.go
Comment (lucifercr07) [inline internal/eval/store_eval.go:1]: @vanshavenger Shall we add unit/integration for this?

**Review by lucifercr07 — CHANGES_REQUESTED** [review]
LGTM, please add tests.

File: internal/eval/store_eval.go
Comment (vanshavenger) [reply internal/eval/store_eval.go:1]: Adding Integration test so that it will be tested against all cases. 
PR Coming in Sometime.

**Comment by vanshavenger** [comment]
@lucifercr07 added tests that were missing and now it is consistent with the DICE DB Sets docs too and Also with redis.

**Review by lucifercr07 — APPROVED** [review]

**Pull request merged**